File: clojure/raindrops/src/raindrops.clj
(ns raindrops)

(defn pling [n]
  (when (== 0 (mod n 3)) "Pling"))

(defn plang [n]
  (when (== 0 (mod n 5)) "Plang"))

(defn plong [n]
  (when (== 0 (mod n 7)) "Plong"))

(defn convert [n]
  (let [result (str (pling n) (plang n) (plong n))]
    (if (empty? result)
      (str n)
      result)))

File: julia/raindrops/raindrops.jl
function raindrops(number)
    s = ""
    number % 3 == 0 && (s *= "Pling")
    number % 5 == 0 && (s *= "Plang")
    number % 7 == 0 && (s *= "Plong")
    s == "" ? string(number) : s
end
